### PR TITLE
Change filter sensitive add distance SENSITIVE_AREA_INTERSECTION_MARGIN

### PR DESCRIPTION
**Pull request by LePetitTim**

### CODE REVIEW

File: geotrek/api/v2/filters.py [hunk 1]
```diff
@@ -411,6 +413,10 @@ def filter_queryset(self, request, queryset, view):
         types = request.GET.get('types')
         if types:
             qs = qs.filter(type__in=types.split(','))
+        trek = request.GET.get('trek', None)
+        if trek is not None:
+            t = Trek.objects.get(pk=trek)
```
Comment (Chatewgne) [inline geotrek/api/v2/filters.py:418]: Use `filter` instead of `get` to avoid exceptions when bad trek ID is provided

**Comment by camillemonchicourt** [comment]
Not sure to understand changelog. This is a filter added in API v2 ? On which route ?

**Comment by LePetitTim** [comment]
> Not sure to understand changelog. This is a filter added in API v2 ? On which route ?

Is it better with : 
- Use distance from setting SENSITIVE_AREA_INTERSECTION_MARGIN in sensitive area filter `trek`

File: geotrek/api/v2/filters.py (hunk 1 above)
Comment (LePetitTim) [reply geotrek/api/v2/filters.py:419]: Done

**Comment by camillemonchicourt** [comment]
Yes, but should mention that it is in API v2, as we could understand that it is in Geotrek-admin.

PS : Just saw that Sensitive area route as two similar filters that can bring confusion : "trek" and "near_trek"...

**Review by Chatewgne — APPROVED** [review]

**Pull request merged**